src/style.css

```diff
@@ -25,6 +26,12 @@ body,
     color: var(--sub-title-color);
 }
 
+.desktop-navigation-bar .avatar,
+.mobile-navigation-bar .avatar {
+    margin: 0 0.6rem;
```

Comment:
0.6rem can be in px

Reply:
I am following the practice to use px only for the border radius, to make it appear the same on all font sizes of the root.